### ISSUE TITLE
[CI] MixedClusterEsqlSpecIT test {row.StaticCidrMatch #2} failing

### ISSUE THREAD

**Issue by maxhniebergall** [elastic/elasticsearch#105423]
**Build scan:**
https://gradle-enterprise.elastic.co/s/ikhzrvc4evbi2/tests/:x-pack:plugin:esql:qa:server:mixed-cluster:v8.12.2%23javaRestTest/org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT/test%20%7Brow.StaticCidrMatch%20%232%7D

**Reproduction line:**
```
./gradlew ':x-pack:plugin:esql:qa:server:mixed-cluster:v8.12.2#javaRestTest' -Dtests.class="org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT" -Dtests.method="test {row.StaticCidrMatch #2}" -Dtests.seed=CF1CF768F0C4888 -Dtests.bwc=true -Dtests.locale=ar-YE -Dtests.timezone=Atlantic/Bermuda -Druntime.java=21
```

**Applicable branches:**
main

**Reproduces locally?:**
Didn't try

**Failure history:**
[Failure dashboard for `org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT#test {row.StaticCidrMatch #2}`](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Brow.StaticCidrMatch%20%232%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure excerpt:**
```
org.elasticsearch.client.ResponseException: method [POST], host [http://[::1]:35229], URI [/_query?format=json&pretty=true&error_trace=true], status line [HTTP/1.1 500 Internal Server Error]
Warnings: [No limit defined, adding default limit of [500]]
{
  "error" : {
    "root_cause" : [
      {
        "type" : "ql_illegal_argument_exception",
        "reason" : "Should not fold expression",
        "stack_trace" : "org.elasticsearch.xpack.ql.QlIllegalArgumentException: Should not fold expression\n\tat org.elasticsearch.xpack.ql.expression.Expression.fold(Expression.java:82)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.lambda$planRow$6(LocalExecutionPlanner.java:499)\n\tat java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)\n\tat java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1708)\n\tat java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)\n\tat java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)\n\tat java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)\n\tat java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)\n\tat java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)\n\tat java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)\n\tat java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planRow(LocalExecutionPlanner.java:499)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:213)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planLimit(LocalExecutionPlanner.java:560)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:203)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planEval(LocalExecutionPlanner.java:410)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:193)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planOutput(LocalExecutionPlanner.java:280)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:227)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:170)\n\tat org.elasticsearch.xpack.esql.plugin.ComputeService.runCompute(ComputeService.java:312)\n\tat org.elasticsearch.xpack.esql.plugin.ComputeService.execute(ComputeService.java:160)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecuteForked$3(TransportEsqlQueryAction.java:110)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:178)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)\n\tat org.elasticsearch.xpack.ql.util.ActionListeners.lambda$map$0(ActionListeners.java:24)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.ql.util.ActionListeners.lambda$map$0(ActionListeners.java:24)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.ql.util.ActionListeners.lambda$map$0(ActionListeners.java:24)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.lambda$preAnalyze$5(EsqlSession.java:170)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.preAnalyzeIndices(EsqlSession.java:207)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.lambda$preAnalyze$6(EsqlSession.java:168)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListener$4.onResponse(ActionListener.java:344)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.support.RefCountingListener.finish(RefCountingListener.java:121)\n\tat org.elasticsearch.base@8.12.2-SNAPSHOT/org.elasticsearch.core.AbstractRefCounted$1.closeInternal(AbstractRefCounted.java:118)\n\tat org.elasticsearch.base@8.12.2-SNAPSHOT/org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:70)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.support.RefCountingRunnable.close(RefCountingRunnable.java:112)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.support.RefCountingListener.close(RefCountingListener.java:114)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.preAnalyze(EsqlSession.java:178)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.analyzedPlan(EsqlSession.java:143)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.optimizedPlan(EsqlSession.java:300)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.physicalPlan(EsqlSession.java:308)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.optimizedPhysicalPlan(EsqlSession.java:316)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.execute(EsqlSession.java:113)\n\tat org.elasticsearch.xpack.esql.execution.PlanExecutor.esql(PlanExecutor.java:67)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.doExecuteForked(TransportEsqlQueryAction.java:104)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecute$0(TransportEsqlQueryAction.java:87)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:95)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"
      }
    ],
    "type" : "ql_illegal_argument_exception",
    "reason" : "Should not fold expression",
    "stack_trace" : "org.elasticsearch.xpack.ql.QlIllegalArgumentException: Should not fold expression\n\tat org.elasticsearch.xpack.ql.expression.Expression.fold(Expression.java:82)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.lambda$planRow$6(LocalExecutionPlanner.java:499)\n\tat java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)\n\tat java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1708)\n\tat java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)\n\tat java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)\n\tat java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)\n\tat java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)\n\tat java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)\n\tat java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)\n\tat java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planRow(LocalExecutionPlanner.java:499)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:213)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planLimit(LocalExecutionPlanner.java:560)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:203)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planEval(LocalExecutionPlanner.java:410)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:193)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planOutput(LocalExecutionPlanner.java:280)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:227)\n\tat org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:170)\n\tat org.elasticsearch.xpack.esql.plugin.ComputeService.runCompute(ComputeService.java:312)\n\tat org.elasticsearch.xpack.esql.plugin.ComputeService.execute(ComputeService.java:160)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecuteForked$3(TransportEsqlQueryAction.java:110)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:178)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)\n\tat org.elasticsearch.xpack.ql.util.ActionListeners.lambda$map$0(ActionListeners.java:24)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.ql.util.ActionListeners.lambda$map$0(ActionListeners.java:24)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.ql.util.ActionListeners.lambda$map$0(ActionListeners.java:24)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.lambda$preAnalyze$5(EsqlSession.java:170)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.preAnalyzeIndices(EsqlSession.java:207)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.lambda$preAnalyze$6(EsqlSession.java:168)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionListener$4.onResponse(ActionListener.java:344)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.support.RefCountingListener.finish(RefCountingListener.java:121)\n\tat org.elasticsearch.base@8.12.2-SNAPSHOT/org.elasticsearch.core.AbstractRefCounted$1.closeInternal(AbstractRefCounted.java:118)\n\tat org.elasticsearch.base@8.12.2-SNAPSHOT/org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:70)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.support.RefCountingRunnable.close(RefCountingRunnable.java:112)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.support.RefCountingListener.close(RefCountingListener.java:114)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.preAnalyze(EsqlSession.java:178)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.analyzedPlan(EsqlSession.java:143)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.optimizedPlan(EsqlSession.java:300)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.physicalPlan(EsqlSession.java:308)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.optimizedPhysicalPlan(EsqlSession.java:316)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.execute(EsqlSession.java:113)\n\tat org.elasticsearch.xpack.esql.execution.PlanExecutor.esql(PlanExecutor.java:67)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.doExecuteForked(TransportEsqlQueryAction.java:104)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecute$0(TransportEsqlQueryAction.java:87)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:95)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)\n\tat org.elasticsearch.server@8.12.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"
  },
  "status" : 500
}

  at __randomizedtesting.SeedInfo.seed([CF1CF768F0C4888:84A5F0AC21F02570]:0)
  at org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT.row.StaticCidrMatch(row.csv-spec:340)
  at org.elasticsearch.client.RestClient.convertResponse(RestClient.java:351)
  at org.elasticsearch.client.RestClient.performRequest(RestClient.java:317)
  at org.elasticsearch.client.RestClient.performRequest(RestClient.java:292)
  at org.elasticsearch.xpack.esql.qa.rest.RestEsqlTestCase.performRequest(RestEsqlTestCase.java:800)
  at org.elasticsearch.xpack.esql.qa.rest.RestEsqlTestCase.performRequest(RestEsqlTestCase.java:796)
  at org.elasticsearch.xpack.esql.qa.rest.RestEsqlTestCase.runEsqlSync(RestEsqlTestCase.java:617)
  at org.elasticsearch.xpack.esql.qa.rest.EsqlSpecTestCase.runEsql(EsqlSpecTestCase.java:162)
  at org.elasticsearch.xpack.esql.qa.rest.EsqlSpecTestCase.doTest(EsqlSpecTestCase.java:140)
  at org.elasticsearch.xpack.esql.qa.rest.EsqlSpecTestCase.test(EsqlSpecTestCase.java:128)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
  at java.lang.reflect.Method.invoke(Method.java:580)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.junit.rules.RunRules.evaluate(RunRules.java:20)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.junit.rules.RunRules.evaluate(RunRules.java:20)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.elasticsearch.test.cluster.local.DefaultLocalElasticsearchCluster$1.evaluate(DefaultLocalElasticsearchCluster.java:47)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at org.junit.rules.RunRules.evaluate(RunRules.java:20)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1583)

```

**Comment by maxhniebergall** [comment]
may have been fixed by https://github.com/elastic/elasticsearch/commit/8272b3a5dabcdf3df197f53f6de0743649e9fdcb

**Comment by alex-spies** [comment]
> may have been fixed by [8272b3a](https://github.com/elastic/elasticsearch/commit/8272b3a5dabcdf3df197f53f6de0743649e9fdcb)

Agree, that should have done the trick. Thanks for pointing it out!
